submissions/migrations/0003_submission_null_FK.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('submissions', '0002_auto_20151119_0913'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='submission',
+            name='student_item',
+            field=models.ForeignKey(to='submissions.StudentItem', null=True, default=1),
+            preserve_default=False
+        ),
+    ]
```

Comment:
@maxrothman Here's some data from running this migration on my devstack with ~1400 rows of data, the read-replica indicates ~670,000 on production

https://gist.github.com/efischer19/b56b9d14768de28cf612